### PR TITLE
Ecdsa private/public key generation

### PR DESCRIPTION
**Pull request by akonior**

### CODE REVIEW

File: zkmarek/test/test_standard.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+from math import sqrt
+from unittest import TestCase
+
+from zkmarek.crypto.ec_affine import ECAffine
+from zkmarek.crypto.standard import Secp256
+from zkmarek.crypto.weierstrass_curve import Secp256k1
+
+
+class TestStandard(TestCase):
+    def test_init(self):
```
Comment (marekkirejczyk) [inline zkmarek/test/test_standard.py:10]: Missing: `test_init_invalid_generator`

File: zkmarek/test/test_standard.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        )
+
+    def test_generate_secret_key(self):
+        for i in range(1000):
+            secrete_key = Secp256.generate_secret_key()
+            sqrt_n = sqrt(Secp256.group_order)
+            self.assertTrue(secrete_key >= sqrt_n)
+            self.assertTrue(secrete_key < Secp256.group_order - sqrt_n)
+
+    def test_generate_secret_keys_are_different(self):
```
Comment (marekkirejczyk) [inline zkmarek/test/test_standard.py:29]: Rename: `test_generate_secret_keys_unique`

**Review by marekkirejczyk — APPROVED** [review]

File: zkmarek/test/test_standard.py (hunk 2 above)
Comment (akonior) [reply zkmarek/test/test_standard.py]: Renamed

File: zkmarek/test/test_standard.py (hunk 1 above)
Comment (akonior) [reply zkmarek/test/test_standard.py:10]: Invalid generator test case added

**Pull request merged**